pkg/v1/providers/tests/Makefile

```diff
@@ -0,0 +1,11 @@
+# Copyright 2021 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+export ACK_GINKGO_RC ?= true
+
+all: unit
+
+unit:
+	ginkgo unit
+
+.PHONY: unit
```

Comment:
Looks like `unit_suite_test.go` has been integrated with CI.

```
=== RUN   TestUnit
Running Suite: Ytt Rendering Unit Test Suite
============================================
Random Seed: 1639009506
Will run 49 of 49 specs

#@data/values
---
VSPHERE_INSECURE: true
PROVIDER_TYPE: vsphere
VSPHERE_FOLDER: ds0
IS_WINDOWS_WORKLOAD_CLUSTER: true
TKG_CLUSTER_ROLE: management
VSPHERE_USERNAME: user_blah
VSPHERE_PASSWORD: pass_1234
VSPHERE_SERVER: vmware-tanzu.com
VSPHERE_DATACENTER: vmware-tanzu-dc.com
VSPHERE_RESOURCE_POOL: myrp
VSPHERE_SSH_AUTHORIZED_KEY: ssh-rsa AAAA...+M7Q== vmware-tanzu.local
TKG_DEFAULT_BOM: tkg-bom-v1.4.0.yaml
KUBERNETES_RELEASE: v1.21.2---vmware.1-tkg.1

•••••••••••••••••••••••••••••••••••••••••••••••••
Ran 49 of 49 Specs in 16.969 seconds
SUCCESS! -- 49 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestUnit (16.98s)
PASS
ok  	github.com/vmware-tanzu/tanzu-framework/pkg/v1/providers/tests/unit	17.000s
```

Could you please check if your newly added tests are part of those 49 specs? This could be done by adding some log msgs and run `make` from your local. 

Since Github CI triggers unit tests from root makefile already (without leveraging this newly introduced makefile), would it make sense to update root makefile to leverage this file to run unit tests? Or at least let's update the outdated readme under `pkg/v1/providers/tests` ?

Reply:
I see, the t-f github CI invokes the unit test here https://github.com/vmware-tanzu/tanzu-framework/blob/59e0d9b82b25faf85cab40e0c1123f75977bd005/Makefile#L473 with `cd <unit_test_dir> && go test -v ./`. So it looks like the Makefile in the unit test folder is no longer needed since they do the same. I will delete this Makefile from the commit.

Reply:
Makefile removed from the unit test dir.